docs/proposals/2170-kubeflow-training-v2/README.md

```diff
@@ -819,49 +824,51 @@ orchestration (e.g. using Kubernetes admission webhooks or custom clients).
 In the future, we can add more parameters if we find use-cases when it is required.
 
 ```golang
-type PodSpecOverride struct {
-	// Name of the training replica in the training runtime template to override
+type PodSpecOverrides struct {
```

Comment:
```suggestion
type PodSpecOverride struct {
```

Could we use the singular name instead of the plural since we used to select the singular name for field typed?
We can image the following fields:

```golang
PodSpecOverrides []PodSpecOverride `json:"podSpecOverrides"` 
```

vs

```golang
PodSpecOverrides []PodSpecOverrides `json:"podSpecOverrides"` 
```